saltstack/base/salt/salt-bootstrap/init.sls

```diff
@@ -1,7 +1,7 @@
 install_saltbootstrap:
   archive.extracted:
     - name: /usr/sbin/
-    - source: https://github.com/hortonworks/salt-bootstrap/releases/download/v0.12.0/salt-bootstrap_0.12.0_Linux_x86_64.tgz
+    - source: https://github.com/hortonworks/salt-bootstrap/releases/download/v0.12.0/salt-bootstrap_0.12.1_Linux_x86_64.tgz
```

Comment:
this link is not valid: `https://github.com/hortonworks/salt-bootstrap/releases/download/v0.12.0/salt-bootstrap_0.12.1_Linux_x86_64.tgz`
replace with v0.12.1 the part after download

Reply:
Thanks for checking it.

Reply:
Fixed